feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/blockhain/calls/ExtrinsicBuilderExt.kt

```diff
@@ -125,6 +125,32 @@ fun ExtrinsicBuilder.parachainScheduleRevokeDelegation(candidate: String): Extri
     )
 }
 
+fun ExtrinsicBuilder.parachainConfirmRevokeDelegation(
+    candidateId: AccountId,
+    delegatorId: AccountId
+): ExtrinsicBuilder {
+    return call(
+        "ParachainStaking", "execute_delegation_request",
+        mapOf(
+            "delegator" to delegatorId,
+            "candidate" to candidateId
+        )
+    )
+}
+
+fun ExtrinsicBuilder.confirmRevokeDelegation(
+    candidateId: AccountId,
+    delegatorId: AccountId
+): ExtrinsicBuilder {
+    return call(
+        "Staking", "execute_delegation_request",
```

Comment:
Staking or ParachainStaking? if Parachain collapse this two fun's into one fun ExtrinsicBuilder.parachainExecuteDelegationRequest(...)